spancontext.go

```diff
@@ -246,6 +267,13 @@ func (c *SpanContext) SetDestinationService(service DestinationServiceSpanContex
 	c.model.Destination = &c.destination
 }
 
+func (c *SpanContext) SetServiceTarget(service ServiceTargetSpanContext) {
```

Comment:
Can you please add comments for the new method and type?

Should we add a deprecation notice to `SpanContext.SetDestinationService` too?

Reply:
Good catch! Added :+1: